src/services/Elements.php

```diff
@@ -3241,17 +3248,71 @@ private function _propagateElement(
 
         // Save it
         $siteElement->setScenario(Element::SCENARIO_ESSENTIALS);
+
+        // for Entries and Categories: validate element against "live" scenario across all sites,
+        // if element is to be set to live/pending/enabled
+        if ($crossSiteValidate && ($siteElement instanceof Entry || $siteElement instanceof Category)) {
```

Comment:
We shouldn’t be making a special case for entries or categories here. The feature should be extended to all element types.

Reply:
ok, pushing an adjustment